lively.collab/comments/commentMorph.js

```diff
@@ -184,7 +184,9 @@ export class CommentMorph extends Morph {
   initializeCommentIndicator () {
     this.commentIndicator = new CommentIndicator(this, this.comment, this.referenceMorph);
     this.commentIndicator.fontColor = this.comment.isResolved() ? Color.rgb(174, 214, 241) : Color.rgb(241, 196, 15);
-    if (CommentBrowser.isOpen()) this.showCommentIndicator();
+    if (CommentBrowser.isOpen() && (this.comment.isResolved() == CommentBrowser.instance.resolvedFilter)) {
```

Comment:
I am not a fan of the name `resolvedFilter`. What about `isInResolvedMode` or `resolvedFilterActive`?